deliaioanac/assignment2/PrintAncestorsUnitTests.cpp

```diff
@@ -0,0 +1,66 @@
+#include "printAncestors.cpp"
+#include <gtest/gtest.h>
+#include <vector>
+
+namespace {
+    class PrintAncestorsTest : public ::testing::Test {
+        protected:
+            BinaryTree tree;
+
+            virtual void SetUp() {
+                /*
+                 * Constructing the following tree :
```

Comment:
Thanks for the visual comments! Really makes it easier to understand the test cases.

Out of curiousity, are you using any Structure-to-ASCII converters to generate these trees? Or are you doing it manually?

Reply:
To be honest, my inspiration was Leetcode ( https://leetcode.com/ ). It is where I learned algorithmics two years ago. When I started writing the tests for this function, I remembered how much their visual aid helped me debug problems, so I thought it could also help you when looking over the code :) I constructed it by hand, but I will certainly look into Structure-to-ASCII converters. I never heard of them before, but they sound like they could be very useful.

Reply:
I see :) I've tried using online structure-to-ASCII converters before but I've found them to be quite difficult to use for tree/graph-like structures, just looks ugly and messy (maybe I haven't learned how to use them properly!). So I usually also do these by hand :)